include/mbgl/ios/MGLMapView.h

```diff
@@ -1,6 +1,10 @@
 #import <UIKit/UIKit.h>
 #import <CoreLocation/CoreLocation.h>
 
+#import "MGLTypes.h"
```

Comment:
Per https://github.com/mapbox/mapbox-gl-native/issues/1090 we can probably move the location-related stuff here instead of including this.